frontend/src/components/TimerComponent/PopUp.js

```diff
@@ -0,0 +1,33 @@
+import * as React from "react";
+import Box from "@material-ui/core/Box";
+
+const style = {
+  position: "absolute",
+  bottom: "0%",
+  right: "0%",
+  width: "20%",
+  bgcolor: "red",
+  border: "2px solid #000",
+  boxShadow: 24,
+  p: 4,
+};
+
+function PopUp({ seconds }) {
+  return (
+    <div>
```

Comment:
Not sure this div is needed either. Could try replacing with a React fragment?
relevant docs: https://react.dev/reference/react/Fragment